README.md

```diff
@@ -34,14 +34,22 @@ Edit this document to include your answers after each question. Make sure to lea
 
 1. What is the difference between an adaptive website and a fully responsive website?
 
+`Adaptive design works with percentages and it is mostly that everything adaptes to the screen size of the device, reponsive design in the other hands response to the actions of the screen size, so if the viewport of the screen size changes, responsive design is going to change as well.`
+
 2. Describe what it means to be mobile first vs desktop first.
 
+`I understand this as, we first design the mobile version first, so we can use DRY later on in the other versions of the design.`
+
 3. What does `font-size: 62.5%` in the `html` tag do for us when using `rem` units?
 
```

Comment:
when using rem units, by having a `font-size: 62.5%` , it sets a ratio up so that 1rem is now equal to 10px.. so using responsive rem units, if you were going for a familiar 16px font size, that would become 1.6 rem :)